particles.html

```diff
@@ -114,23 +116,25 @@
     // Create buffers for render pass
     ///////////////////////////////////
 
-    const [vertexBuffer, vertexBufferMap] = device.createBufferMapped({
+    const vertexBuffer = device.createBuffer({
         size: 32,
-        usage: GPUBufferUsage.VERTEX | GPUBufferUsage.COPY_DST
+        usage: GPUBufferUsage.VERTEX,

```

Comment:
Happy to add the `COPY_DST` back in, but it seems like the `getMappedRange()` approach works without needing it.